css/src/atomics/flex.scss

```diff
@@ -0,0 +1,423 @@
+//Align content
+
+.align-content-flex-start {
+	align-content: flex-start !important;
+}
+
+.align-content-flex-end {
+	align-content: flex-end !important;
+}
+
+.align-content-center {
```

Comment:
I think we can use some sass control flow to help us set these names in stone: https://sass-lang.com/documentation/at-rules/control/each

```scss
@use 'sass:map';

$flex-atomics: (
	'justify-content': (
		flex-start,
		flex-end,
		center,
		space-around,
		space-around
	),
	'align-items': (
		flex-start,
		flex-end,
		center,
		baseline,
		stretch
	),
	'align-self': (
		flex-start,
		flex-end,
		center
	),
	'flex-direction': (
		column,
		column-reverse,
		row,
		row-reverse
	),
	'flex-wrap': (
		wrap
	)
) !default;


@each $key in map.keys($flex-atomics) {
	@each $value in map.get($flex-atomics, $key) {
		.#{$key}-#{$value} {
			#{$key}: $value !important;
			@debug 'Should create: #{$key}-#{$value}';
		}
	}
}
```